guide/14-deep-learning/object-detection.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
```

Comment:
<p>Can you add some examples here? Perhaps from the pavement cracks or some other sample?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/559/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
Could you help me with what exactly to put? The link to the notebook or some part from the notebook?

Reply:
<p>Since the text is a bit abstract, you can add a part of the notebook (graphics, code snippets that show how predict video function is called) and hyperlink to the notebook for full workflow.</p><p> </p>

Reply:
Would the hyperlink suffice? Since it would contain the code and the full workflow. Otherwise it would be repetition of the same thing.